### PR TITLE
CI update: e2e Testnet Paseo

### LINKED ISSUE #1898: Switch Testnet CI Tests to Paseo
Blocked pending Frequency Paseo Testnet launch.

Task:
- [ ] Replace https://github.com/LibertyDSNP/frequency/blob/main/.github/workflows/rococo.yml with a `testnet-paseo` version.

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to add e2e CI tests against Testnet Paseo.

Tests passing in CI: https://github.com/LibertyDSNP/frequency/actions/runs/8636089744

Closes #1898 

# Checklist
- [x] Tests added

## Things Done
- Added new paseo e2e tests
- Update e2e to use Node 20
- Switch to use tsx instead of ts-node in both e2e and api-augment
- Update packages for both e2e and api-augment

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Release
 run-name: Cut Release ${{github.event.inputs.release-version || github.ref_name}}
 concurrency:
   group: ${{github.workflow}}-${{github.ref}}
-  cancel-in-progress: true
+  cancel-in-progress: false
```
Comment (wilwade) [inline .github/workflows/release.yml:6]: Releases should not be stopped by the next release

File: .github/workflows/rococo.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
         run: |
           version=${{env.TAG_FROM_UI}}
           echo "Release version entered in UI: $version"
-          regex='^v([0-9]+)\.(0|([1-9][0-9]+))\.(0|([1-9][0-9]+))(-rc[1-9][0-9]*)?$'
+          regex='^v([0-9]+)\.(0|([1-9][0-9]*))\.(0|([1-9][0-9]*))(-rc[1-9][0-9]*)?$'
```
Comment (wilwade) [inline .github/workflows/rococo.yml:28]: Bug fix from the release flow

File: e2e/scaffolding/env.ts [hunk 3]
```diff
@@ -4,20 +4,80 @@ export const verbose = process.env.VERBOSE_TESTS === 'true' || process.env.VERBO
...
+  switch (process.env.CHAIN_ENVIRONMENT) {
+    case CHAIN_ENVIRONMENT.ROCOCO_TESTNET:
+    case CHAIN_ENVIRONMENT.PASEO_TESTNET:
+    case CHAIN_ENVIRONMENT.ROCOCO_LOCAL:
+    case CHAIN_ENVIRONMENT.PASEO_LOCAL:
+      return true;
+  }
+  return false;
+}
+
```
Comment (wilwade) [inline e2e/scaffolding/env.ts:35]: All the schemas below for Testnet Paseo are all deployed.

**Review by enddynayn — APPROVED** [review]
👍 great!

File: e2e/.load-test.mocharc.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "extension": ["ts"],
   "parallel": false,
   "require": ["scaffolding/globalHooks.ts", "scaffolding/rootHooks.ts", "scaffolding/extrinsicHelpers.ts"],
-  "loader": "ts-node/esm",
+  "import": "tsx/esm",
```
Comment (JoeCap08055) [inline e2e/.load-test.mocharc.json:6]: 💯

File: Makefile [hunk 5]
```diff
@@ -247,7 +247,7 @@ build-testnet-release:
 build-mainnet-release:
 	cargo build --locked --features  frequency --release
 
-.PHONY: test, e2e-tests, e2e-tests-serial, e2e-tests-only, e2e-tests-load, e2e-tests-load-only, e2e-tests-rococo, e2e-tests-rococo-local
+.PHONY: test, e2e-tests, e2e-tests-serial, e2e-tests-only, e2e-tests-load, e2e-tests-load-only, e2e-tests-rococo, e2e-tests-rococo-local, e2e-tests-testnet-paseo, e2e-tests-paseo-local
```
Comment (JoeCap08055) [inline Makefile:250]: Is `e2e-test-paseo-local` supposed to work out of the box in a local dev environment? Or what are the instructions;  it seems to expect Frequency to be running already...

File: scripts/run_e2e_tests.sh [hunk 6]
```diff
@@ -70,6 +70,18 @@ case "${CHAIN}" in
...
+        PROVIDER_URL="ws://127.0.0.1:9944"
+        NPM_RUN_COMMAND="test:relay"
+        CHAIN_ENVIRONMENT="paseo-local"
+    ;;
+    "paseo_testnet")
+        PROVIDER_URL="wss://0.rpc.amplica.io"
+        NPM_RUN_COMMAND="test:relay"
+        CHAIN_ENVIRONMENT="paseo-testnet"
+
+        read -p "Enter the seed phrase for the Frequency Rococo account funding source: " FUNDING_ACCOUNT_SEED_PHRASE
```
Comment (JoeCap08055) [inline scripts/run_e2e_tests.sh:83]:
```suggestion
        read -p "Enter the seed phrase for the Frequency Paseo account funding source: " FUNDING_ACCOUNT_SEED_PHRASE
```

File: Makefile (hunk 5 above)
Comment (wilwade) [reply Makefile:250]: Paseo local is special. It does expect Frequency to already be running as setting up a full local relay chain is not easy.

**Review by JoeCap08055 — APPROVED** [review]
Lgtm--might be nice at some point to add docs on how to run e2e locally against testnet, but that can be for later

File: e2e/.node-version [hunk 7]
```diff
@@ -1 +1 @@
-18.16.0
+20.11.1
```
Comment (mattheworris) [inline e2e/.node-version:1]: Any reason not to use the current 20.x LTS version from April 10, 2024: `20.12.2`?

**Review by mattheworris — APPROVED** [review]
- Read through changes.
- One non-blocking comment.

🚢 it!

File: e2e/.node-version (hunk 7 above)
Comment (wilwade) [reply e2e/.node-version:1]: Most of the other stuff uses 20.11.1? That's the only reason. It was a copy paste.

**Pull request merged**